control_plane/storcon_cli/src/main.rs

```diff
@@ -77,6 +77,11 @@ enum Command {
         #[arg(long)]
         scheduling: Option<ShardSchedulingPolicyArg>,
     },
+    /// Print details about a particular node, including all shards attached to it.
+    NodeAttached {
```

Comment:
I wonder if one could expose it as an optional `node` param of `Tenants` instead.